css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  font-family: Lato, serif;
+  background-color: rgb(25, 150, 218);
+  color: white;
+}
+
+section {
+  max-width: 2004px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h2 {
+  font-size: 62px;
+  margin-top: 120px;
+  letter-spacing: 0.92px;
+  font-weight: 300;
+}
+
+h3 {
+  font-size: 43px;
+  font-weight: 300;
+}
+
+p {
+  font-size: 29px;
+  margin: 0 80px;
+  color: rgb(221, 221, 221);
+}
+
+.white-circle {
+  background-color: white;
+  width: 240px;
+  height: 240px;
+  border: 2px solid rgb(221, 221, 221);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 120px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+img {
+  background-color: white;
+  width: calc(124% / 2.4);
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
I would separate these into two different classes. 

Theoretically, what if you want another element in a `row` but your don't want it to be centered? I would create a separate class for your `justify-content: center` named `justify-center` and apply that as well as the `row` class to the element you want to center it's flex children.

Reply:
Good point! Though after playing around with things I realize that was a holdover from when I thought I was going to shrink my columns and have space between them. Now that they together take up with whole row the justify-content is unnecessary, so I deleted it.